### DISCUSSION TITLE
Cookie Topic Change

### DISCUSSION THREAD

**Discussion by drichardson** [tosdr/edit.tosdr.org#1002] [Ideas]
FYI, I made a big change to an important topic today, but I'm not where where to discuss/announce this so I'm putting it here.

https://edit.tosdr.org/topics/25

I renamed the *Cookie* to *Trackers*, in order to explicitly include any kind of tracking technology, not just HTTP cookies.

Here's the new info:

### Title
Trackers

### Sub-title
All kinds of tracking, including cookies, session storage, and beacons.

### Description
Covers any technology that allows services to track (identify over time, place, and between devices) a user. These technologies include but are not limited to:

- HTTP Cookies
- IP geolocation
- Browser Session Storage
- iOS Advertising IDs
- Device Fingerprinting
- Data de-anonymization/Data re-identification

**Comment by JesseWeinstein** [comment]
This makes a lot of sense. We need to clean up the individual cases, too; I haven't been approving points in those cases because I couldn't make sense of what the criteria were.

**Comment by drichardson** [comment]
By the way, there is overlap between [Anonymity and Tracking](https://edit.tosdr.org/topics/26) and [Trackers](https://edit.tosdr.org/topics/25) (formerly Cookies).

For example, *Anonymity and Tracking* has *The service uses third party tracking pixels to track social media conversion*, which I think definitely belongs in tracking.

*Anonymity* and *Tracking* are closely related, but definitely not the same. You can track a user and  build a profile of them from clickstream interaction without knowing who they are.

I'm going to change *Anonymity and Tracking* to just *Anonymity*, and move the tracking specific Cases from *Anonymity* topic to *Tracking* topic.

**Comment by drichardson** [comment]
OK all done, check out:

- [Anonymity](https://edit.tosdr.org/topics/26)
- [Trackers](https://edit.tosdr.org/topics/25)

I also moved one Case under [Personal Data](https://edit.tosdr.org/topics/41): [App required for this service requires broad device permissions](https://edit.tosdr.org/cases/322), as this had more to do with levels of data access vs being anonymous or not (though again, there is definitely overlap between Personal Data and Anonymity).
